server/resource_controller.go

```diff
@@ -88,6 +113,65 @@ func (rc *ResourceController) IndexHandler(rw http.ResponseWriter, r *http.Reque
 	json.NewEncoder(rw).Encode(&bundle)
 }
 
+func generatePagingLinks(query search.Query, total uint32) ([]models.BundleLinkComponent, error) {
+	links := make([]models.BundleLinkComponent, 0, 5)
+	values := query.NormalizedQueryValues(false)
+	options := query.Options()
+	count := uint32(options.Count)
+	offset := uint32(options.Offset)
+
+	// First create the base URL for paging
+	host, err := os.Hostname()
```

Comment:
We have had issues simply using os.Hostname() in the past. As discussed with @cmoesel, we should probably use the hostname that was in the request. In other cases we have tried to resolve the hostname's IP, and defaulted to localhost if the IP could not be resolved. [See here](https://github.com/intervention-engine/ie/blob/master/server.go#L35)